src/styles/card.css

```
@@ -0,0 +1,86 @@
+.card {
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  background-color: var(--color-fff);
+  border: 1px solid var(--color-F3F3F3);
+  border-radius: 5px;
+}
+
+.body__card {
+  width: 200px;
+
+}
+
```

```suggestion
.body__card {
  width: 200px;
}

```
.body__card is an element of body BEM block so is shouldn't be in this file